Added vaishnavi_muchukota user profile and portrait

vmuchukota opened the pull request.

lgilpin reviewed (changes requested):
I think is a repeat of #122 and #121

lgilpin reviewed (changes requested):
You should only commit the `.md` file and your portrait file.

nawzanina committed:
Ignore local dev auditor page for Nazanin

nawzanina committed:
Reset docs directory to match main

nawzanina committed:
Update member profile for Nazanin

nawzanina committed:
Update Hugo config and add CI workflow for Hugo 0.152.2

nawzanina committed:
Guard research authors partial when authors are not defined

nawzanina committed:
Remove docs/auditor/nazanin from .gitignore

lgilpin committed:
Merge pull request #141 from aiea-lab/Nazanin_auditor

Add Nazanin auditor profile

Commit:
Adding new publications

nawzanina committed:
Add two new publications

collin-garrison committed:
Adding Collin user profile

vseslav-k committed:
Add Vseslav Kazakov's member profile

vseslav-k committed:
Rename vseslav to vseslav.md

vseslav-k committed:
added Vee pfp

CoffeeCoder1015 committed:
Adding George user profile

calebjames02 committed:
Merge pull request #153 from collin-garrison/collin_auditor

Add Collin auditor profile

CoffeeCoder1015 committed:
Edit: added missing research interests

calebjames02 committed:
Merge pull request #155 from CoffeeCoder1015/ggong_auditor

Adding George auditor profile

vseslav-k committed:
Delete content/member/vseslav.md

vseslav-k committed:
Add Vseslav Kazakov's profile information

calebjames02 committed:
Merge pull request #154 from vseslav-k/main

Added Vee's profile

rudrachavda committed:
Adding rudra user profile

rkonchit committed:
Ron's user profile

Commit:
Adding Jacob Wei user profile

Commit:
Adding Anesh Kondor user profile

KunwarpalSingh10 committed:
Adding Kunwarpal Singh user profile

calebjames02 committed:
Merge pull request #160 from aneshkondor/AneshKondor_auditor

Adding Anesh Kondor user profile

KunwarpalSingh10 committed:
Adding Kunwarpal Singh user profile

justinrosales committed:
Adding Justin user profile

rkonchit committed:
Adding fix

Commit:
updates

calebjames02 committed:
Merge pull request #162 from justinrosales/justinr_auditor

Adding Justin user profile

calebjames02 committed:
Merge pull request #161 from KunwarpalSingh10/kunwarpalsingh_auditor

Adding Kunwarpal Singh user profile

calebjames02 committed:
Merge pull request #158 from r-konchit/ron_auditor

Ron's user profile

calebjames02 committed:
Merge pull request #159 from jjacobhw/jacob_auditor

Adding Jacob Wei user profile

Commit:
Add auditor profile for Anurag

Commit:
Add auditor profile for Anurag with default avatar

Commit:
Adding Sree Gudapati user profile

Commit:
Adding auditor profile for Anurag

rudrachavda committed:
added bio and interests as requested

asenthilvasan committed:
Adding ashwin_senthilvasan user profile

Commit:
Adding mahir_camci user profile

Commit:
roshni g

rudrachavda committed:
added image file, forgot to add it in original commit

Commit:
Adding Omer Boztepe  user profile

cyu139 committed:
Adding Crystal Yu user profile

Commit:
Adding Anurag's user profile

cyu139 committed:
Adding Crystal Yu user profile

darthnithin committed:
Adding Nithin user profile

IdoHaiby committed:
Adding idohaiby user profile

calebjames02 committed:
Merge pull request #169 from cyu139/crystal_auditor

Task 1 Crystal Yu - Adding Crystal Yu user profile

calebjames02 committed:
Merge pull request #157 from aiea-lab/rudra_auditor

Adding rudra user profile

calebjames02 committed:
Merge pull request #164 from asenthilvasan/ashwin_senthilvasan_auditor

Adding ashwin_senthilvasan user profile

calebjames02 committed:
Merge pull request #165 from camcimahir/main

Adding mahir_camci user profile

Commit:
fixed github link for omer boztepe profile (Task 1)

OmerBoztepe committed:
Merge branch 'aiea-lab:main' into omer_boztepe_auditor

calebjames02 committed:
Merge pull request #168 from OmerBoztepe/omer_boztepe_auditor

Adding Omer Boztepe  user profile

Commit:
Updating github link

calebjames02 committed:
Merge pull request #163 from sgudapa1-ui/sree_gudapati_auditor

Adding Sree Gudapati user profile

nryee2005 committed:
Adding Nathan Yee user profile

calebjames02 committed:
Merge pull request #173 from aiea-lab/nryee_auditor

Adding Nathan Yee user profile

IdoHaiby committed:
Adding idohaiby user research interests

IdoH04 committed:
Merge branch 'aiea-lab:main' into idohaiby_auditor

Commit:
adding user profile for Rami Al Habash

Profese committed:
Added Joshua Sojan user profile

Profese committed:
Merge pull request #1 from Profese/joshua_sojan_auditor

Added Joshua Sojan user profile

Commit:
Removed excess file

darthnithin committed:
fix github link

calebjames02 committed:
Merge pull request #171 from darthnithin/nithin_auditor

Adding Nithin user profile

calebjames02 committed:
Update idohaiby.md

Adding quotation marks around interests.

calebjames02 committed:
Merge pull request #172 from IdoH04/idohaiby_auditor

Adding idohaiby user profile

calebjames02 committed:
Merge pull request #170 from aiea-lab/Anurag_auditor

Add auditor profile for Anurag Nangunoori

Commit:
Add Roshni Gundavelli as auditor

asvinod committed:
Adding Ashwin Vinod user profile

woozoo21 committed:
Adding Sashreek Rewatkar user profile

Chr0no9 committed:
Adding Marco Nasirpour user profile

calebjames02 committed:
Merge pull request #166 from aiea-lab/roshnigundavelli_auditor

Add Roshni Gundavelli as auditor

calebjames02 committed:
Merge pull request #174 from Rami-habash/main

adding user profile for Rami Al Habash

calebjames02 committed:
Merge pull request #176 from Profese/main

Add Joshua Sojan profile

calebjames02 committed:
Merge pull request #178 from asvinod/ashwin_vinod_auditor

Adding Ashwin Vinod user profile

calebjames02 committed:
Rename can-large-language-models-explain-themselves?-a-study-of-llm-generated-self-explanations-2023.md to can-large-language-models-explain-themselves-a-study-of-llm-generated-self-explanations-2023.md

calebjames02 committed:
Rename right-this-way-can-vlms-guide-us-to-see-more-to-answer-questions?-2024.md to right-this-way-can-vlms-guide-us-to-see-more-to-answer-questions-2024.md

lgilpin committed:
Merge pull request #184 from calebjames02/main

Update file paths to remove question marks

gaurijain21 committed:
Updating GauriJain auditor user profile

woozoo21 committed:
updated sashreek rewatkar github link

Chr0no9 committed:
Fill in research interests for Marco auditor profile

Anish888-coder committed:
Added Anish Bansal profile

calebjames02 committed:
Merge pull request #180 from woozoo21/sashreek_auditor

Adding Sashreek Rewatkar user profile

calebjames02 committed:
Merge pull request #186 from Anish888-coder/anishbansal_auditor

Added Anish Bansal profile

calebjames02 committed:
Merge pull request #185 from gaurijain21/main

Updating GauriJain auditor user profile

calebjames02 committed:
Update marco.md

Put quotation marks around interests

calebjames02 committed:
Update marco.md

Fixed date joined

calebjames02 committed:
Merge pull request #181 from Chr0no9/marco_auditor

Adding Marco Nasirpour user profile

Ena-AlexBrush committed:
Add auditor profile for Chanchal Mukeshsingh

Haardhik7177 committed:
Adding Haardhik user profile: Winter 2026

Haardhik7177 committed:
Added interests

calebjames02 committed:
Merge pull request #188 from Ena-AlexBrush/chanchal_auditor

Add auditor profile for Chanchal Mukeshsingh

Commit:
Adding my user profile for Raunak Anwar

calebjames02 committed:
Merge pull request #191 from Raunakulous69/Raunak_Anwar_auditor

Adding my user profile for Raunak Anwar

calebjames02 committed:
Merge pull request #190 from aiea-lab/haardhik_auditor

Haardhik auditor

taniamakhija committed:
Create tania.md

Add Tania auditor profile

taniamakhija committed:
Add files via upload

calebjames02 committed:
update file location of tania.jpg

calebjames02 committed:
Merge pull request #192 from taniamakhija/tania-profile-new

Tania profile new

calebjames02 committed:
Deleting duplicate file

GirishAdithi committed:
Add Adithi Girish auditor profile

Haatim-Ali committed:
Add Haatim Ali auditor profile

serehen committed:
Added sergio user profile

S-Spektrum-M committed:
Adding siddharth_mohanty user profile

calebjames02 committed:
Merge pull request #195 from S-Spektrum-M/siddharth_mohanty_auditor

Adding siddharth_mohanty user profile

calebjames02 committed:
Merge pull request #194 from serehen/main

Added sergio user profile

calebjames02 committed:
Update adithi.md

Updating file path to have it match the uploaded image

calebjames02 committed:
Merge pull request #193 from aiea-lab/adithi_auditor

Add Adithi Girish auditor profile

Haatim-Ali committed:
Adding Haatim Ali user profile

calebjames02 committed:
Merge pull request #196 from 1HawtTamale/Haatim_Ali_Auditor

Adding Haatim Ali user profile

mandyy8 committed:
adding mandy_wu user profile

rmundhra committed:
Adding Riddhi user profile

calebjames02 committed:
Merge pull request #198 from aiea-lab/riddhi_auditor

Adding Riddhi user profile

AJMCS committed:
Added portrait file

AJMCS committed:
Create AJMcDaniel.md with personal details and bio

Added a new markdown file for AJ McDaniel with bio, interests, education, and social links.

calebjames02 committed:
Merge pull request #199 from AJMCS/main

Add AJ McDaniel profile with personal details and bio

lgilpin committed:
Merge pull request #148 from aiea-lab/hugo-ci-only

Add Hugo v0.152.2 CI build workflow and allow Goldmark raw HTML

Commit:
Add Ran Chen profile and fix Tania date format

anthonymfrance committed:
Adding Anthony France profile

asmoly committed:
Adding Alexander Smolyanskiy user profile

calebjames02 committed:
Merge pull request #201 from jchen842-stack/Ran_Chen_auditor

Add Ran Chen auditor profile fix Tania date format

Commit:
Removing the workflow for now

Commit:
Adding new docs to build the new website

lgilpin committed:
Create hugo.yml

Trying to make the automatic hugo builder

lgilpin committed:
Merge pull request #204 from aiea-lab/workflow-patch

Create hugo.yml

lgilpin committed:
Update hugo.yml

Changing to deploy from docs

calebjames02 committed:
Merge branch 'main' into anthony_france_auditor

JonathanWMorris committed:
Added Testing

lgilpin committed:
Merge pull request #205 from aiea-lab/jonathan

Added Testing

JonathanWMorris committed:
Merge branch 'main' into alexander_smolyanskiy_auditor

JonathanWMorris committed:
Merge branch 'main' into anthony_france_auditor

calebjames02 committed:
Merge pull request #202 from anthonymfrance/anthony_france_auditor

Adding Anthony France profile

asmoly committed:
Merge branch 'main' into alexander_smolyanskiy_auditor

utkarshbank committed:
Add auditor profile for Utkarsh Banka and portrait image

asmoly committed:
Change to Adding Alexander Smolyanskiy user profile

utkarshbank committed:
Merge branch 'main' into UtkarshBanka_auditor

asmoly committed:
Merge branch 'alexander_smolyanskiy_auditor' of https://github.com/asmoly/aiea-lab.github.io into alexander_smolyanskiy_auditor

dhonk committed:
Adding daniel_hong user profile

dhonk committed:
Merge branch 'main' into daniel_hong_auditor

e-ember committed:
Adding ember_lu user profile

e-ember committed:
Merge branch 'main' into ember_auditor

calebjames02 committed:
Merge pull request #203 from asmoly/alexander_smolyanskiy_auditor

Adding Alexander Smolyanskiy user profile

dhonk committed:
fixed missing interests

utkarshbank committed:
Clean up social media links in UtkarshBanka.md

Removed unused social media links for Twitter and Google Scholar.

utkarshbank committed:
Rename Utkarsh_B.JPG to Utkarsh_B.jpg

Commit:
Bowie's profile additions

Commit:
Revert "Bowie's profile additions"

This reverts commit 238ff371f5ab1d5345c67c2cb5b2ed417e501ca9.

Commit:
Bowie's auditor profiel

e-ember committed:
Updating image name

e-ember committed:
Merge branch 'ember_auditor' of https://github.com/aiea-lab/aiea-lab.github.io into ember_auditor

e-ember committed:
Merge branch 'main' into ember_auditor

mnarla committed:
Add Mayank Narla to auditor profiles

calebjames02 committed:
Merge pull request #213 from Bochew/Bowie_auditor

Bowie's auditor user profile

calebjames02 committed:
Merge branch 'main' into daniel_hong_auditor

calebjames02 committed:
Merge pull request #209 from aiea-lab/daniel_hong_auditor

Adding daniel_hong user profile

calebjames02 committed:
Merge pull request #206 from utkarshbank/UtkarshBanka_auditor

Add auditor profile for Utkarsh Banka and portrait image

calebjames02 committed:
Merge pull request #211 from aiea-lab/ember_auditor

Adding ember_lu user profile

calebjames02 committed:
Merge pull request #214 from mnarla/Mayanknarla_auditor

Add Mayank Narla to auditor profiles

ndontubo committed:
Add files via upload

Adding navaneeth user profile

ndontubo committed:
Adding navaneeth user profile

ndontubo committed:
Merge branch 'aiea-lab:main' into main

samanyuk committed:
Create SamanyuKumar.md

calebjames02 committed:
Merge pull request #216 from ndontubo/main

adding navaneeth auditor profile

samanyuk committed:
Add files via upload

samanyuk committed:
Merge pull request #1 from samanyuk/new-Samanyu-Kumar-profile

Add Samanyu Kumar profile

samanyuk committed:
Merge branch 'aiea-lab:main' into main

calebjames02 committed:
Update SamanyuKumar.md file path

calebjames02 committed:
Merge pull request #217 from samanyuk/main

New Samanyu Kumar profile

waterball01 committed:
Adding Ruhan Gianchandani user profile

NagaGun committed:
Add Naga auditor profile and portrait

Made-with: Cursor

Sanyab8 committed:
Adding sanya_bhatia  user profile

Daniel-Dubiner committed:
Adding Daniel Dubiner user profile

Daniel-Dubiner committed:
Merge branch 'main' into Daniel_Dubiner_auditor

calebjames02 committed:
Merge pull request #210 from NagaGun/naga_auditor

Naga auditor

mnarla committed:
Update portrait in mayank.md

Sanyab8 committed:
Add new file sanya_b.md

Sanyab8 committed:
Merge branch 'main' into sanya_auditor

AbhyuThoppae committed:
Adding user profile

Sanyab8 committed:
Change portrait file extension from .jpg to .jpeg

waterball01 committed:
Adding Ruhan user profile

waterball01 committed:
Editing Ruhan Gianchandai user profile

AbhyuThoppae committed:
Fixed profile issues

mnarla committed:
Update for mayank.md

Commit:
Add Caitelyn auditor profile

Haatim-Ali committed:
Remove auto-generated docs folder

Haatim-Ali committed:
Add docs to gitignore

Haatim-Ali committed:
Update date in Haatim Ali's bio

Haatim-Ali committed:
Revert "Add docs to gitignore"

This reverts commit c98ca5e72865939b73eefc5cf6bf856a8c5a2478.

Haatim-Ali committed:
Revert "Remove auto-generated docs folder"

This reverts commit 766998f47cd79af098a0cbdfc86e41b035880f2c.

Haatim-Ali committed:
Fix typos in Haatim Ali bio

calebjames02 committed:
Merge pull request #220 from mnarla/patch-1

Update portrait in mayank.md

calebjames02 committed:
Merge pull request #221 from aiea-lab/sanya_auditor

Adding sanya_bhatia  user profile

calebjames02 committed:
Merge pull request #218 from aiea-lab/ruhan_auditor

Adding Ruhan Gianchandani user profile

calebjames02 committed:
Merge pull request #222 from AbhyuThoppae/abhyuthoppae_auditor

Adding user profile

calebjames02 committed:
Merge pull request #226 from 1HawtTamale/Haatim_Ali_Auditor

Adding Haatim Ali auditor profile

calebjames02 committed:
Merge pull request #212 from crispi-cloud/caitelyn_auditor

Add Caitelyn auditor profile

Daniel-Dubiner committed:
Added Daniel Dubiner user profile bio and interests

Daniel-Dubiner committed:
Merge branch 'main' into Daniel_Dubiner_auditor

calebjames02 committed:
Merge pull request #219 from aiea-lab/Daniel_Dubiner_auditor

Adding Daniel Dubiner user profile

snraja-ucsc committed:
Adding Surya_Raja User Profile

aryanbhcode committed:
Adding Aryan Bhatia User Profile

calebjames02 committed:
Merge branch 'main' into Surya_Raja_auditor

calebjames02 committed:
Merge pull request #228 from aryanbhcode/AryanB_auditor

Adding Aryan Bhatia User Profile

dylankprice committed:
Adding Dylan Price user profile

snraja-ucsc committed:
resolved comments

calebjames02 committed:
Small formatting update

calebjames02 committed:
Merge pull request #227 from snraja-ucsc/Surya_Raja_auditor

Adding Surya_Raja User Profile

calebjames02 committed:
Merge pull request #230 from aiea-lab/dylanprice_auditor

Adding Dylan Price user profile

Commit:
Adding Lars Voigt-Bui user profile

Commit:
Adding Lars Voigt-Bui user profile

BhoomikaGuptaa committed:
Adding bhoomika user profile

Commit:
Adding Hyerin Chung user profile

calebjames02 committed:
Merge pull request #233 from BhoomikaGuptaa/bhoomika_auditor

Adding bhoomika user profile

calebjames02 committed:
Merge pull request #234 from aiea-lab/hyerin_auditor

Adding Hyerin Chung user profile

Commit:
fixing PR #232

Commit:
removing lines 17-20

calebjames02 committed:
Merge pull request #232 from lars-v-bui/LarsVoigt-Bui_auditor

Lars voigt bui auditor

JonathanWMorris committed:
Updated bio

JonathanWMorris committed:
Changed photo

tbellibe committed:
Add files via upload

tbellibe committed:
Add files via upload

tbellibe committed:
Add files via upload

tbellibe committed:
Delete tanvibelli.jpg

calebjames02 committed:
Merge pull request #236 from tbellibe/main

Tanvi Belli Belli

mandyy8 committed:
added interests to mandy_wu profile

calebjames02 committed:
Merge pull request #197 from mandyy8/mandy_wu_auditor

adding mandy_wu user profile

calebjames02 committed:
Merge branch 'main' into Jonathan

calebjames02 committed:
Merge pull request #235 from aiea-lab/Jonathan

Updated Jonathan Morris Bio and Picture

calebjames02 committed:
Merge branch 'main' into danielrhee_auditor

calebjames02 committed:
Merge pull request #44 from DanielRhee/danielrhee_auditor

Adding Daniel Rhee user Profile

Commit:
Adding Shefali Gokarn auditor profile

calebjames02 committed:
Merge pull request #238 from Shefaligokarn/shefali_auditor

Adding Shefali Gokarn auditor profile

calebjames02 committed:
Merge branch 'main' into suhani_auditor

calebjames02 committed:
Merge pull request #54 from aiea-lab/suhani_auditor

Adding suhani's user profile

lgilpin committed:
Merge branch 'main' into main

calebjames02 committed:
Update lucy_h.md

calebjames02 committed:
Update lucy_h.md

calebjames02 committed:
Update lucy_h.md

calebjames02 committed:
Merge pull request #68 from LucyHan118/main

Adding lucy han user profile

lgilpin committed:
Merge pull request #152 from aiea-lab/Nazanin_auditor

Add two new publications

calebjames02 committed:
Merge branch 'main' into pavithraselvaraj

calebjames02 committed:
Merge pull request #75 from PavithraKandhasamySelvaraj/pavithraselvaraj

Adding Pavithra User profile

calebjames02 committed:
Merge branch 'main' into ssduars1

calebjames02 committed:
Merge pull request #76 from aiea-lab/ssduars1

Add sameeerkashyap

calebjames02 committed:
Merge branch 'main' into pujitha_auditor

calebjames02 committed:
Merge pull request #116 from aiea-lab/pujitha_auditor

Added pujitha user profile
